build.sbt

```diff
@@ -59,4 +59,8 @@ lazy val codecs = (projectMatrix in file("codecs"))
   })
   .jvmPlatform(scalaVersions = supportedScalaVersions)
 
-// See https://www.scala-sbt.org/1.x/docs/Using-Sonatype.html for instructions on how to publish to Sonatype.
+lazy val plugin = (projectMatrix in file("plugin"))
+  .settings(name := "safer-serializer-plugin")
+  .settings(commonSettings)
+  .settings(libraryDependencies ++= Seq(scalaCompiler213, scalaLibrary213, scalaReflect213))
```

Comment:
Ok... so as for now, the plugin is 2.13 only right?

Reply:
2.12 would be useful (even instead of 2.13, if cross-compilation is non-trivial) since as of today, Hydra is still running on 2.12